test-results.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I assume this file snuck in unintentionally?

Reply:
Yes it did